bare-metal/elastic-metal/reference-content/elastic-metal-rv1-advanced-usage.mdx

```diff
@@ -0,0 +1,105 @@
+---
+meta:
+  title: Elastic Metal RV1 advanced usage
+  description: This page explains some advanced ways to use EM-RV1 servers.
+content:
+  h1: Elastic Metal RV1 advanced usage
+  paragraph: This page explains some advanced ways to use EM-RV1 servers.
+tags: elastic-metal server riscv em-rv1
+dates:
+  validation: 2024-03-15
+  posted: 2024-03-15
+categories:
+  - bare-metal
+---
+
+## Boot process
+
+EM-RV1 servers do not support standard UEFI or BIOS boot, therefore the boot
+process might slightly differ from what you are used to on other servers. At
```

Comment:
```suggestion
process might slightly differ from other usual servers. At
```

Reply:
I just rephrased that to "other servers" as I don't think "usual server" would mean the same to everyone.